tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -303,7 +303,7 @@ export class DataTableComponent implements OnDestroy, AfterContentInit {
     return (
       this.selectableColumns &&
       this.selectableColumns.length &&
-      this.contextMenuHeader?.movable
```

Comment:
This line is important. We have pinned columns (e.g. the runs column) which we do not want to move. If we allow columns to be inserted to the left of these columns they will functionally have moved.

Reply:
Good point. Re-added this and updated the corresponding test.